src/eos/eosEngine.js

```diff
@@ -713,17 +713,35 @@ export class EosEngine extends CurrencyEngine {
     if (this.walletInfo.keys.eosOwnerKey) {
       keyProvider.push(this.walletInfo.keys.eosOwnerKey)
     }
-    const signedTx = await this.multicastServers('transaction',
-      edgeTransaction.otherParams.transactionJson,
-      {
-        keyProvider,
-        sign: true,
-        broadcast: true
+    try {
+      const signedTx = await this.multicastServers('transaction',
+        edgeTransaction.otherParams.transactionJson, {
+          keyProvider,
+          sign: true,
+          broadcast: true
+        })
+      edgeTransaction.date = Date.now() / 1000
+      edgeTransaction.txid = signedTx.transaction_id
+      return edgeTransaction
+    } catch (e) {
+      let err = e
+      try {
+        err = JSON.parse(e)
+      } catch (e2) {
+        throw e
       }
-    )
-    edgeTransaction.date = Date.now() / 1000
-    edgeTransaction.txid = signedTx.transaction_id
-    return edgeTransaction
+      if (err.error && err.error.name === 'tx_net_usage_exceeded') {
```

Comment:
Do we want to make these errors official in the Core? They obviously work fine like this for now.